media-sound/snapcast/snapcast-9999.ebuild

```diff
@@ -1,6 +1,6 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
-# $Id$
+# $id$
```

Comment:
OK, I went to find the correct version of the ID line, and it looks like as of February 2017 the line is no longer required. You can go ahead and just delete the ID line.

https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#ebuild-header